{"uid":"732addc5048243e1","name":"test_add_to_cart","fullName":"testcase.test_inventory_page.TestInventoryPage#test_add_to_cart","historyId":"81fdb3ac8f944f2f9f0960c229f3fef9","time":{"start":1668550178495,"stop":1668550178495,"duration":0},"status":"skipped","statusMessage":"Skipped: already included in purchase test","statusTrace":"('/Users/runner/work/sauce_webtest/sauce_webtest/testcase/test_inventory_page.py', 11, 'Skipped: already included in purchase test')","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1668550178547,"stop":1668550178659,"duration":112},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"driver","time":{"start":1668550178659,"stop":1668550260856,"duration":82197},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: chrome not reachable\nStacktrace:\n0   chromedriver                        0x000000010468b2c8 chromedriver + 4752072\n1   chromedriver                        0x000000010460b463 chromedriver + 4228195\n2   chromedriver                        0x000000010426e98a chromedriver + 440714\n3   chromedriver                        0x000000010425c874 chromedriver + 366708\n4   chromedriver                        0x000000010429a896 chromedriver + 620694\n5   chromedriver                        0x0000000104296046 chromedriver + 602182\n6   chromedriver                        0x0000000104291e3c chromedriver + 585276\n7   chromedriver                        0x00000001042d4c43 chromedriver + 859203\n8   chromedriver                        0x00000001042d474f chromedriver + 857935\n9   chromedriver                        0x00000001042ccb33 chromedriver + 826163\n10  chromedriver                        0x000000010429d9fd chromedriver + 633341\n11  chromedriver                        0x000000010429f051 chromedriver + 639057\n12  chromedriver                        0x000000010465830e chromedriver + 4543246\n13  chromedriver                        0x000000010465ca88 chromedriver + 4561544\n14  chromedriver                        0x00000001046646df chromedriver + 4593375\n15  chromedriver                        0x000000010465d8fa chromedriver + 4565242\n16  chromedriver                        0x00000001046332cf chromedriver + 4391631\n17  chromedriver                        0x000000010467c5b8 chromedriver + 4691384\n18  chromedriver                        0x000000010467c739 chromedriver + 4691769\n19  chromedriver                        0x000000010469281e chromedriver + 4782110\n20  libsystem_pthread.dylib             0x00007ff81ccb94e1 _pthread_start + 125\n21  libsystem_pthread.dylib             0x00007ff81ccb4f6b thread_start + 15\n\n","statusTrace":"  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1128, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/_pytest/fixtures.py\", line 900, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/runner/work/sauce_webtest/sauce_webtest/testcase/conftest.py\", line 22, in driver\n    get_driver = ChromeBrowser().browser\n  File \"/Users/runner/work/sauce_webtest/sauce_webtest/util/browser.py\", line 97, in browser\n    chrome = self._browser(service=self.service, options=self.options)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/chrome/webdriver.py\", line 81, in __init__\n    super().__init__(\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/chromium/webdriver.py\", line 106, in __init__\n    super().__init__(\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 288, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 381, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/remote/webdriver.py\", line 444, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/runner/hostedtoolcache/Python/3.8.14/x64/lib/python3.8/site-packages/selenium/webdriver/remote/errorhandler.py\", line 249, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"afterStages":[],"labels":[{"name":"tag","value":"@pytest.mark.skip('already included in purchase test')"},{"name":"tag","value":"inventory"},{"name":"parentSuite","value":"testcase"},{"name":"suite","value":"test_inventory_page"},{"name":"subSuite","value":"TestInventoryPage"},{"name":"host","value":"Mac-1668549935470.local"},{"name":"thread","value":"3076-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"testcase.test_inventory_page"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["inventory","@pytest.mark.skip('already included in purchase test')"]},"source":"732addc5048243e1.json","parameterValues":[]}